deployer/src/ec2/mod.rs

```diff
@@ -139,115 +140,126 @@
 //!     storage_class: gp2
 //!     binary: /path/to/binary
 //!     config: /path/to/config.conf
+//!     profiling: true
 //!   - name: node2
 //!     region: us-west-2
 //!     instance_type: t4g.small
 //!     storage_size: 10
 //!     storage_class: gp2
 //!     binary: /path/to/binary2
 //!     config: /path/to/config2.conf
+//!     profiling: false
 //! ports:
 //!   - protocol: tcp
 //!     port: 4545
 //!     cidr: 0.0.0.0/0
 //! ```
 
 use serde::{Deserialize, Serialize};
-use std::{net::IpAddr, path::PathBuf};
-use thiserror::Error;
-
-pub mod aws;
-mod create;
-pub mod services;
-pub use create::create;
-mod update;
-pub use update::update;
-mod authorize;
-pub use authorize::authorize;
-mod destroy;
-pub use destroy::destroy;
-pub mod utils;
+use std::net::IpAddr;
+
+cfg_if::cfg_if! {
```

Comment:
This allows anyone that imports `deployer` to create config files to avoid compiling heavyweight `aws` crates.